src/App.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<router-view />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import WalletStore from "./store/WalletStore";
+import { WalletKind } from "./types";
+declare var AlgoSigner: any; // eslint-disable-line
+
+export default defineComponent({
+	name: "App",
+	components: {},
+	data() {
+		return {
+			loading: true,
+		};
+	},
```

Comment:
according to our prettier config, we should use 1tab, not 1 space. Please run prettier - it will fix it.